js-pkg/react/src/main.tsx

```diff
@@ -31,18 +31,25 @@ export function useAwareness(): Awareness {
   return yjsCtx.provider.awareness
 }
 
-export function usePresence<T extends Record<string, any>>(): [
-  Map<number, T>,
-  (presence: T) => void,
-] {
+type UsePresenceOptions = {
+  excludeSelf?: boolean
+}
+
+export function usePresence<T extends Record<string, any>>(
+  options?: UsePresenceOptions,
+): [Map<number, T>, (presence: T) => void] {
   const awareness = useAwareness()
   const [presence, setPresence] = useState<Map<number, T>>(new Map())
 
+  const excludeSelf = options?.excludeSelf ?? true
```

Comment:
Minor, but if the default is to exclude self, maybe this option could be called `includeSelf`?

Reply:
good call, I like the principle that bools should be oriented such that the default is false.